charts/eoapi/profiles/core.yaml

```diff
@@ -0,0 +1,217 @@
+        - postgres
+      options: "CREATEDB CREATEROLE"
+      password:
+        type: AlphaNumeric
+
+######################
+# PGSTAC BOOTSTRAP
+######################
+pgstacBootstrap:
+  enabled: true
+  settings:
+    # Disable sample data for production
+    loadSamples: false
+
+    # Production-optimized PgSTAC settings
+    pgstacSettings:
+      queue_timeout: "10 minutes"
+      use_queue: "false"
+      update_collection_extent: "true"
+      context: "auto"
+      context_estimated_count: "100000"
+      context_estimated_cost: "100000"
+      context_stats_ttl: "1 day"
+
+    resources:
+      requests:
+        cpu: "512m"
+        memory: "1024Mi"
+      limits:
+        cpu: "1024m"
+        memory: "2048Mi"
+
+######################
+# CORE API SERVICES
+######################
+# Only stable, production-ready services
+stac:
+  enabled: true
+  ingress:
+    enabled: true
+    path: "/stac"
+  autoscaling:
+    enabled: false
+  settings:
+    resources:
+      requests:
+        cpu: "512m"
+        memory: "1024Mi"
+      limits:
+        cpu: "1024m"
+        memory: "2048Mi"
+    envVars:
+      HOST: "0.0.0.0"
+      PORT: "8080"
+      WEB_CONCURRENCY: "4"
+      # Production settings
+      STAC_FASTAPI_DEBUG: "False"
+      STAC_FASTAPI_CORS_ORIGINS: '["*"]'
+
+raster:
+  enabled: true
+  ingress:
+    enabled: true
+    path: "/raster"
+  autoscaling:
+    enabled: false
+  settings:
+    resources:
+      requests:
+        cpu: "512m"
+        memory: "2048Mi"
+      limits:
+        cpu: "1024m"
+        memory: "4096Mi"
+    envVars:
+      # GDAL performance settings
+      GDAL_CACHEMAX: "200"
+      GDAL_DISABLE_READDIR_ON_OPEN: "EMPTY_DIR"
+      GDAL_INGESTED_BYTES_AT_OPEN: "32768"
+      GDAL_HTTP_MERGE_CONSECUTIVE_RANGES: "YES"
+      GDAL_HTTP_MULTIPLEX: "YES"
+      GDAL_HTTP_VERSION: "2"
+      GDAL_SKIP: "VRT"
+      PYTHONWARNINGS: "ignore"
+      VSI_CACHE: "TRUE"
+      VSI_CACHE_SIZE: "5000000"
+      # Uvicorn settings
+      HOST: "0.0.0.0"
+      PORT: "8080"
+      WEB_CONCURRENCY: "4"
+      # Production settings
+      TITILER_DEBUG: "False"
+
+vector:
+  enabled: true
+  ingress:
+    enabled: true
+    path: "/vector"
+  autoscaling:
+    enabled: false
+  settings:
+    resources:
+      requests:
+        cpu: "256m"
+        memory: "512Mi"
+      limits:
+        cpu: "512m"
+        memory: "1024Mi"
+    envVars:
+      TIPG_CATALOG_TTL: "300"
+      TIPG_DEBUG: "False"
+      HOST: "0.0.0.0"
+      PORT: "8080"
+      WEB_CONCURRENCY: "4"
+
+######################
+# DISABLED SERVICES
+######################
+# Experimental and optional services disabled
+multidim:
+  enabled: false
+
+browser:
+  enabled: false
+
+docServer:
+  enabled: true
+
+eoapi-notifier:
+  enabled: false
+
+knative:
+  enabled: false
+
+monitoring:
+  metricsServer:
+    enabled: false
+  prometheus:
+    enabled: false
+  prometheusAdapter:
+    enabled: false
+
+observability:
+  grafana:
+    enabled: false
```

Comment:
I'd maybe consider these 'production/core'?

Reply:
I would prefer to keep the `core.yaml` rather conservatively small. But it probably makes sense to add a `production.yaml` version with them being enabled. Kind of a "what we would deploy for a partner usually". But maybe in another PR?

`core.yaml` should be more like the things we are confident they are _stable_. Potentially also give a different warning on these when there are breaking changes than on the others.